app/src/main/java/com/lastaoutdoor/lasta/viewmodel/DiscoverScreenViewModel.kt

```diff
@@ -110,7 +111,7 @@ constructor(
     }
   }
 
-  private fun activitiesToMarkers(activities: List<Activity>): List<Marker> {
```

Comment:
is it normal to change this method from private to public? Is that the desired result? Because if we're not calling this function in any other place, maybe we should keep it private :)

Reply:
Yeah I hesitated but it's just so that I'm able to test it in `DiscoveryScreenViewModelTest` to provide additional line coverage. It's weird but a private function cannot be tested BUT counts as uncovered lines